packages/token-lists/src/buildList.ts

```diff
@@ -4,24 +4,33 @@ import { TokenList } from "@uniswap/token-lists";
 import { version } from "../package.json";
 import pancakeswapDefault from "./tokens/pancakeswap-default.json";
 import pancakeswapExtended from "./tokens/pancakeswap-extended.json";
+import pancakeswapTop100 from "./tokens/pancakeswap-top-100.json";
 
 const lists = {
   "pancakeswap-default": {
     list: pancakeswapDefault,
-    name: "PancakeSwap Default Token List",
+    name: "PancakeSwap Default",
     keywords: ["pancakeswap", "default"],
     logoURI:
       "https://assets.trustwalletapp.com/blockchains/smartchain/assets/0x0E09FaBB73Bd3Ade0a17ECC321fD13a19e81cE82/logo.png",
     sort: false,
   },
   "pancakeswap-extended": {
     list: pancakeswapExtended,
-    name: "PancakeSwap Extended Token List",
```

Comment:
This is fixed in the https://github.com/pancakeswap/pancake-toolkit/pull/130 PR
Let's merge it so there is less stuff out-of-date in this PR